nilearn/mass_univariate/tests/test_utils.py

```diff
@@ -268,3 +269,16 @@ def test_t_score_with_covars_and_normalized_design_withcovar(rng):
     # compute t-scores with linalg or statmodels
     ref_score = get_tvalue_with_alternative_library(var1, var2, covars)
     assert_array_almost_equal(own_score, ref_score)
+
+
+@pytest.mark.parametrize("two_sided_test", [True, False])
+@pytest.mark.parametrize("dh", [1 / 49, "auto", 0.1])
+@pytest.mark.parametrize("arr3d", [np.ones((10, 11)), _rng().random((10, 11))])
+def test_return_score_threshs(arr3d, two_sided_test, dh):
+    """Check that the range of thresholds to test."""
+    score_threshs = _utils._return_score_threshs(
+        arr3d, dh=dh, two_sided_test=two_sided_test
+    )
+
+    max_score = np.max(np.abs(arr3d)) if two_sided_test else np.max(arr3d)
+    assert np.sum(score_threshs > max_score) == 0
```

Comment:
I checked and the previous implementation failed this test

Reply:
It looks good to me, thank you.